cfgov/jinja2/v1/_includes/organisms/sidebar-breakout.html

```diff
@@ -70,18 +62,18 @@
 
    settings:                An object used to customize the markup.
 
-   settings.heading:        A string containing heading text.
+   settings.heading:        A string containing heading text. ()
```

Comment:
To make things a bit more organized, I used an `options` object in the hero molecule. I think if there's only one option, a parameter is fine, but once you get into multiple, an object to contain them is a lot easier to follow.